features/step_definitions/package_json_steps.coffee

```diff
@@ -4,11 +4,13 @@ path = require 'path'
 
 module.exports = ->
 
-  @Given /^I am in a node project$/, (done) ->
-    fsExtra.outputJson path.join(@tmpDir, 'package.json'), name: '123', done
+  @Given /^I work on a Node.js project$/, (done) ->
```

Comment:
Okay. Just to illustrate my thought, you could build a node project though without a package.json. If you're only using the built in tools to create an app and don't publish to npm. Granted not publishing to npm but using it for dependencies is entirely possible, but not required. I know I'm constructing a very rare case, but still a possible one